website/build.gradle

```diff
@@ -49,7 +49,7 @@ task createDockerContainer(type: Exec) {
     return standardOutput.toString().trim()
   }
   commandLine '/bin/bash', '-c',
-    "docker create -v $project.rootDir:$dockerWorkDir -u \$(id -u):\$(id -g) $dockerImageTag"
+    "docker create -v $project.rootDir:$dockerWorkDir -u \$(id -u):\$(id -g) -p 127.0.0.1:4000:4000 $dockerImageTag"
```

Comment:
I'm still learning Docker, so just checking: this binding will work for local execution and on Jenkins?